src/main/settings.py

```diff
@@ -0,0 +1,128 @@
+"""Django settings.
+
+Generated by 'django-admin startproject' using Django 5.0.6.
+
+For more information on this file, see:
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see:
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+# Built-in libraries:
+from pathlib import Path
+import os
+
+# Third-party dependencies:
+from dotenv import load_dotenv
+
+# Set environment variables.
+load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production.
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret.
+SECRET_KEY = "django-insecure-e9^@hkx*@hqh()&qrle@0z5awy$ao_erfzd1)_4s9a_@cz3^tz"
+
+# SECURITY WARNING: Don't run with debug turned on in production.
+DEBUG = True
+
+ALLOWED_HOSTS: list[str] = []
+
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    "rest_framework",
+    "affiliations",
+]
+
+MIDDLEWARE = [
+    "django.middleware.security.SecurityMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+]
+
+ROOT_URLCONF = "main.urls"
+
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [f"{BASE_DIR}/affiliations/templates"],
+        "APP_DIRS": True,
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
+                "django.contrib.auth.context_processors.auth",
+                "django.contrib.messages.context_processors.messages",
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = "main.wsgi.application"
+
+# Database:
+# https://docs.djangoproject.com/en/5.0/ref/settings/#databases
+DATABASES = {
+    "default": {
+        "ENGINE": f'django.db.backends.{os.getenv("AFFILS_DB_ENGINE")}',
```

Comment:
In the .env file, there is a variable ```export AFFILS_DB_ENGINE="django.db.backends.postgresql"``` and in this settings file, ```django.db.backends``` is appended to the beginning, which throws this error:
django.core.exceptions.ImproperlyConfigured: 'django.db.backends.django.db.backends.postgresql' isn't an available database backend.

Changing to ```"ENGINE": os.getenv("AFFILS_DB_ENGINE"),``` fixed this issue for me.

Reply:
I think the key in the `.env` file should be "postgresql" rather than "django.db.backends.postgresql". This is a bit pedantic, but I think it's more accurate and from the perspective of the person filling out the `.env` file I think it makes more sense.

Reply:
Sounds good!

Reply:
Addressed in https://github.com/ClinGen/stanford-affils/pull/82/commits/b6fa4d09f81018ff97b0f9d73a0205f8ef9ce846.